lms/static/comments.js

```diff
@@ -1,4 +1,5 @@
 const DEFAULT_COMMENTED_LINE_COLOR = '#fab3b0';
+const STUDENT_COMMENTED_LINE_COLOR = '#b2eefa';
```

Comment:
Not such good selection for a color - can you try something a bit more dul? Maybe around #cdffd8. WDYT?